docs/logql.md

```diff
@@ -0,0 +1,159 @@
+# LogQL: Log Query Language
+
+Loki comes with its very own language for querying logs called *LogQL*. LogQL
+can be considered a distributed `grep` with labels for extra querying control.
```

Comment:
`with labels for extra querying control` maybe is more clear `with labels for filtering`?